benchmarks/linear_algebra/blas/level2/gemv/gemv_generator.cpp

```diff
@@ -0,0 +1,129 @@
+#include <tiramisu/tiramisu.h>
+
+using namespace tiramisu;
+
+
+/**
+ * Benchmark for BLAS GEMV
+ *     out = a*A*x + b*y
+ *  
+ *
```

Comment:
Can you please remove extra new lines? Keep only one new line.

Reply:
Sorry for that. Done.